### DISCUSSION TITLE
Issue with getHeaderParameter

### DISCUSSION THREAD

**Discussion by lpatwell** [zbateson/mail-mime-parser#246] [General]
I've been using the following code for reading my emails. It was working fine until I got the error below. I tried a few options to fix it but none works. Any idea how to make it work?

CODE:
if ( null === $message->getTextPart()->getHeaderParameter('content-type', 'charset'))
        {
            $message->getTextPart()->setCharsetOverride('utf-8');
        }

ERROR:
Fatal error: Uncaught Error: Call to a member function getHeaderParameter() on null in /myscript.php:67 Stack trace: #0 {main} thrown in /myscript.php on line 67  (line 67 is the if statement)

**Comment by zbateson** [comment]
The error's telling you "$message->getTextPart()" is null (there's no text part in the message).
